packages/@glimmer/syntax/lib/builders.ts

```diff
@@ -572,6 +569,85 @@ export default {
   },
 };
 
+const locationBuilders = {
+  loc: buildLoc,
+  pos: buildPosition,
+};
+
+const modernBuilders = {
+  MustacheStatement: legacyBuilders.mustache,
+  BlockStatement: legacyBuilders.block,
+  PartialStatement: legacyBuilders.partial,
+  CommentStatement: legacyBuilders.comment,
+  MustacheCommentStatement: legacyBuilders.mustacheComment,
+  ElementNode: legacyBuilders.element,
+  ElementModifierStatement: legacyBuilders.elementModifier,
+  AttrNode: legacyBuilders.attr,
+  TextNode: legacyBuilders.text,
+  SubExpression: legacyBuilders.sexpr,
+  PathExpression: legacyBuilders.path,
+  ConcatStatement: legacyBuilders.concat,
+  Hash: legacyBuilders.hash,
+  HashPair: legacyBuilders.pair,
+  Literal: legacyBuilders.literal,
+  Program: legacyBuilders.program, // returning type - Template
+  Block: legacyBuilders.blockItself,
+  Template: legacyBuilders.template,
+  StringLiteral: legacyBuilders.string,
+  BooleanLiteral: legacyBuilders.boolean,
+  NumberLiteral: legacyBuilders.number,
+  UndefinedLiteral: legacyBuilders.undefined,
+  NullLiteral: legacyBuilders.null,
+};
+
+const modernLowercasedBuilders = {
+  mustacheStatement: legacyBuilders.mustache,
+  blockStatement: legacyBuilders.block,
+  partialStatement: legacyBuilders.partial,
+  commentStatement: legacyBuilders.comment,
+  mustacheCommentStatement: legacyBuilders.mustacheComment,
+  elementNode: legacyBuilders.element,
+  elementModifierStatement: legacyBuilders.elementModifier,
+  attrNode: legacyBuilders.attr,
+  textNode: legacyBuilders.text,
+  subExpression: legacyBuilders.sexpr,
+  pathExpression: legacyBuilders.path,
+  concatStatement: legacyBuilders.concat,
+  hash: legacyBuilders.hash,
+  hashPair: legacyBuilders.pair,
+  literal: legacyBuilders.literal,
+  program: legacyBuilders.program, // returning type - Template
+  block: legacyBuilders.blockItself,
```

Comment:
When merged, this will conflict with the much older `builders.block`

https://github.com/glimmerjs/glimmer-vm/blob/8b92c2c1a71cb11f1b81e5bbce3f4397d06fbfe6/packages/%40glimmer/syntax/lib/builders.ts#L544

Not sure that we can get away with the merging behaviors because of this 🤔

Reply:
we can apply `Capitalized` case or crate named export for updated builders, instead of 'default'